src/mscorlib/src/System/TimeZoneInfo.cs

```diff
@@ -319,24 +319,58 @@ private class OffsetAndRule {
             }
         }
 
+#if PLATFORM_UNIX
+        // The rules we use in Unix cares mostly about the start and end dates but doesn’t fill the transition start and end info. 
+        // as the rules now is public, we should fill it properly so the caller doesn’t have to know how we use it internally 
+        // and can use it as it is used in Windows
 
-        // ---- SECTION: public methods --------------*
+        private AdjustmentRule [] GetFilledRules()
+        {
+            Contract.Assert(m_adjustmentRules != null, "m_adjustmentRules expected to be not null");
+            AdjustmentRule [] rules = new AdjustmentRule[m_adjustmentRules.Length];
```

Comment:
just wondering why spacing here is not good?

---
In reply to: [87495205](https://github.com/dotnet/coreclr/pull/8074#discussion_r87495205) [](ancestors = 87495205)